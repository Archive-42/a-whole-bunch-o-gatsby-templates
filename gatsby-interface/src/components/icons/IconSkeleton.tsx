import React from "react"
import { IconSize, IconSkeletonProps } from "./types"

const iconHeightBySize: Record<IconSize, string> = {
  inherit: `1em`,
  xxsmall: `16px`,
  xsmall: `20px`,
  small: `24px`,
  medium: `32px`,
  large: `40px`,
}

export default function IconSkeleton({
  iconName,
  size = `small`,
  style,
  applyColorToStroke = true,
  ...rest
}: IconSkeletonProps) {
  return (
    <svg
      preserveAspectRatio="xMidYMid meet"
      height={iconHeightBySize[size]}
      width={iconHeightBySize[size]}
      viewBox="0 0 24 24"
      fill={applyColorToStroke ? `none` : `currentColor`}
      stroke={applyColorToStroke ? `currentColor` : `none`}
      strokeWidth="1"
      fillRule="evenodd"
      data-testid={`icon-${iconName}`}
      style={{
        verticalAlign: `middle`,
        ...(style || {}),
      }}
      aria-hidden
      {...rest}
    />
  )
}
