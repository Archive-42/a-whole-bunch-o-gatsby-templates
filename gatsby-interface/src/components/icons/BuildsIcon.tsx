import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"

export default function BuildsIcon(props: IconProps) {
  return (
    <IconSkeleton {...props} iconName="BuildsIcon">
      <rect
        y="0.999935"
        width="9.54724"
        height="9.54724"
        transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8699 11.55)"
        stroke="currentColor"
        strokeWidth="2"
      />
      <rect
        y="0.999935"
        width="9.54724"
        height="9.54724"
        transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8664 7)"
        stroke="currentColor"
        strokeWidth="2"
      />
      <rect
        y="0.999935"
        width="9.54724"
        height="9.54724"
        transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8699 2)"
        fill="white"
        stroke="currentColor"
        strokeWidth="2"
        strokeMiterlimit="11.4737"
      />
    </IconSkeleton>
  )
}
