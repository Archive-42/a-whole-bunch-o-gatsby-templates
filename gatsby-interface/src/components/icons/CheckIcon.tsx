import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"

export default function CheckIcon(props: IconProps) {
  return (
    <IconSkeleton {...props} iconName="CheckIcon" applyColorToStroke={false}>
      <path d="M9.3 16.7l-3.1-4.3A1 1 0 0 1 7.8 11l3.7 3.9c.6.6.6 1.6 0 2.1-.7.5-1.7.4-2.2-.3z" />
      <path d="M9.6 17c-.7-.6-.7-1.5-.2-2.2l7.3-8a1 1 0 0 1 1.6 1.3l-6.6 8.6c-.5.7-1.5.8-2.1.3z" />
      <path d="M9.7 13c.4.5.7.8 1.1.3.5-.5-.1 1.3-.1 1.3l-1.5-.5.5-1z" />
    </IconSkeleton>
  )
}
