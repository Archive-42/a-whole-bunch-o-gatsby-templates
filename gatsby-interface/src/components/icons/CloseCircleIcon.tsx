import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"

export default function CloseCircleIcon(props: IconProps) {
  return (
    <IconSkeleton
      {...props}
      iconName="CloseCircleIcon"
      applyColorToStroke={false}
    >
      <circle cx="12" cy="12" r="10" fill="currentColor" />
      <path
        d="M16.6666 8.27331L15.7266 7.33331L12 11.06L8.27331 7.33331L7.33331 8.27331L11.06 12L7.33331 15.7266L8.27331 16.6666L12 12.94L15.7266 16.6666L16.6666 15.7266L12.94 12L16.6666 8.27331Z"
        fill="white"
      />
    </IconSkeleton>
  )
}
