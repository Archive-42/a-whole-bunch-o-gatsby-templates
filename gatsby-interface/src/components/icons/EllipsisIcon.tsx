import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"

export default function EllipsisIcon(props: IconProps) {
  return (
    <IconSkeleton {...props} iconName="EllipsisIcon" applyColorToStroke={false}>
      <filter id="EllipsisIcon__filter" x="0" y="0" width="100%" height="100%">
        <feComponentTransfer>
          <feFuncA type="linear" slope="0.1"></feFuncA>
        </feComponentTransfer>
      </filter>
      <circle
        cx="12"
        cy="12"
        r="10"
        style={{ filter: `url(#EllipsisIcon__filter)` }}
      />
      <circle cx="6.5" cy="12" r="2" />
      <circle cx="12" cy="12" r="2" />
      <circle cx="17.5" cy="12" r="2" />
    </IconSkeleton>
  )
}
