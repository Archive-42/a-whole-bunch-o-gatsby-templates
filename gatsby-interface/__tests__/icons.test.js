import React from "react"
import { render } from "@testing-library/react"

import * as icons from "../src/components/icons/icons"

const sortedIconComponentNames = Object.keys(icons).sort()

describe(`icons`, () => {
  sortedIconComponentNames
    // filter out __esModule
    .filter(componentName => typeof icons[componentName] !== `boolean`)
    .forEach(componentName => {
      describe(`<${componentName} />`, () => {
        const IconComponent = icons[componentName]

        it(`renders unchanged`, () => {
          const { container } = render(<IconComponent />)

          expect(container).toMatchSnapshot()
        })
      })
    })
})
