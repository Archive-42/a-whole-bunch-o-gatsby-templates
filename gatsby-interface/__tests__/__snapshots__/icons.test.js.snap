// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons <BlogIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-BlogIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M8.49188 11.265L7.43188 15.508L11.6739 14.447L18.0379 8.083L14.8559 4.901L8.49188 11.265Z"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M21.751 4.37097L22.811 5.43097C22.9504 5.57027 23.061 5.73567 23.1364 5.91772C23.2119 6.09978 23.2507 6.29491 23.2507 6.49197C23.2507 6.68904 23.2119 6.88417 23.1364 7.06622C23.061 7.24827 22.9504 7.41367 22.811 7.55297L19.5 10.863"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M18.038 8.083L21.751 4.371C22.0322 4.0897 22.1901 3.70824 22.1901 3.3105C22.1901 2.91275 22.0322 2.53129 21.751 2.25L20.69 1.189C20.4087 0.90779 20.0272 0.749817 19.6295 0.749817C19.2317 0.749817 18.8502 0.90779 18.569 1.189L14.856 4.9"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M18.75 14.25V21.75C18.75 22.1478 18.592 22.5294 18.3107 22.8107C18.0294 23.092 17.6478 23.25 17.25 23.25H2.25C1.85218 23.25 1.47064 23.092 1.18934 22.8107C0.908035 22.5294 0.75 22.1478 0.75 21.75V6.75C0.75 6.35218 0.908035 5.97064 1.18934 5.68934C1.47064 5.40804 1.85218 5.25 2.25 5.25H9.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;

exports[`icons <BuildsIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-BuildsIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <rect
      height="9.54724"
      stroke="currentColor"
      stroke-width="2"
      transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8699 11.55)"
      width="9.54724"
      y="0.999935"
    />
    <rect
      height="9.54724"
      stroke="currentColor"
      stroke-width="2"
      transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8664 7)"
      width="9.54724"
      y="0.999935"
    />
    <rect
      fill="white"
      height="9.54724"
      stroke="currentColor"
      stroke-miterlimit="11.4737"
      stroke-width="2"
      transform="matrix(0.866044 0.499967 -0.866044 0.499967 12.8699 2)"
      width="9.54724"
      y="0.999935"
    />
  </svg>
</div>
`;

exports[`icons <CheckCircleIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-CheckCircleIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <circle
      cx="12"
      cy="12"
      r="10"
    />
    <path
      d="M9.87727 15.7413L7.38952 12.3449C7.1394 11.983 7.20585 11.4901 7.54289 11.2073L7.54289 11.2073C7.87994 10.9245 8.37685 10.9446 8.6899 11.2538L11.6026 14.2935C12.0767 14.7618 12.0453 15.5364 11.5349 15.9647L11.5349 15.9647C11.0244 16.3931 10.2561 16.2895 9.87727 15.7413Z"
      fill="white"
    />
    <path
      d="M10.0811 15.9563C9.56249 15.5212 9.51156 14.7414 9.96912 14.2425L15.748 7.94176C16.0414 7.6218 16.5348 7.58958 16.8674 7.86865L16.8674 7.86865C17.2 8.14773 17.2539 8.63917 16.9898 8.98374L11.7882 15.7689C11.3764 16.3062 10.5996 16.3914 10.0811 15.9563L10.0811 15.9563Z"
      fill="white"
    />
    <path
      d="M10.2123 12.8631C10.5034 13.2125 10.7169 13.4454 11.0856 13.0378C11.4544 12.6302 10.9692 14.0664 10.9692 14.0664L9.76593 13.6782L10.2123 12.8631Z"
      fill="white"
    />
  </svg>
</div>
`;

exports[`icons <CheckIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-CheckIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M9.3 16.7l-3.1-4.3A1 1 0 0 1 7.8 11l3.7 3.9c.6.6.6 1.6 0 2.1-.7.5-1.7.4-2.2-.3z"
    />
    <path
      d="M9.6 17c-.7-.6-.7-1.5-.2-2.2l7.3-8a1 1 0 0 1 1.6 1.3l-6.6 8.6c-.5.7-1.5.8-2.1.3z"
    />
    <path
      d="M9.7 13c.4.5.7.8 1.1.3.5-.5-.1 1.3-.1 1.3l-1.5-.5.5-1z"
    />
  </svg>
</div>
`;

exports[`icons <CloseCircleIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-CloseCircleIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <circle
      cx="12"
      cy="12"
      fill="currentColor"
      r="10"
    />
    <path
      d="M16.6666 8.27331L15.7266 7.33331L12 11.06L8.27331 7.33331L7.33331 8.27331L11.06 12L7.33331 15.7266L8.27331 16.6666L12 12.94L15.7266 16.6666L16.6666 15.7266L12.94 12L16.6666 8.27331Z"
      fill="white"
    />
  </svg>
</div>
`;

exports[`icons <EcommerceIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-EcommerceIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M3.75 5.25001C3.74961 4.96756 3.80495 4.6878 3.91286 4.42677C4.02076 4.16574 4.17912 3.92857 4.37884 3.72885C4.57857 3.52912 4.81574 3.37077 5.07677 3.26286C5.33779 3.15496 5.61755 3.09961 5.9 3.10001"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M0.75 4.687C0.75 3.64284 1.16479 2.64145 1.90312 1.90312C2.64145 1.16479 3.64284 0.75 4.687 0.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M10.5001 16.5H7.6731C7.39125 16.5008 7.11717 16.4077 6.89422 16.2353C6.67127 16.0628 6.51217 15.821 6.4421 15.548L4.5421 9.061C4.49636 8.87508 4.49359 8.68119 4.534 8.49405C4.57441 8.3069 4.65694 8.13143 4.77532 7.98095C4.89369 7.83047 5.04481 7.70895 5.21718 7.62561C5.38955 7.54228 5.57864 7.49932 5.7701 7.5H18.9801C19.1719 7.49866 19.3614 7.54116 19.5343 7.62424C19.7071 7.70732 19.8587 7.82878 19.9775 7.97937C20.0962 8.12996 20.179 8.30569 20.2195 8.49314C20.26 8.68059 20.2571 8.87482 20.2111 9.061L19.9001 10.116"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M16.4999 7.5L15.6709 3.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M9 7.5L9.828 3.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M23.135 23.25L23.235 22.128C23.301 21.4017 23.1001 20.6762 22.67 20.0873C22.2399 19.4983 21.6099 19.0862 20.898 18.928L16.726 18V12C16.726 11.6022 16.5679 11.2206 16.2866 10.9393C16.0053 10.658 15.6238 10.5 15.226 10.5C14.8282 10.5 14.4466 10.658 14.1653 10.9393C13.884 11.2206 13.726 11.6022 13.726 12V21.75L12.026 20.472C11.731 20.2504 11.3659 20.1428 10.9979 20.1688C10.6298 20.1949 10.2836 20.3528 10.0227 20.6137C9.7618 20.8746 9.60383 21.2209 9.5778 21.5889C9.55176 21.9569 9.65942 22.322 9.88098 22.617L10.355 23.25"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;

exports[`icons <EllipsisIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-EllipsisIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <filter
      height="100%"
      id="EllipsisIcon__filter"
      width="100%"
      x="0"
      y="0"
    >
      <fecomponenttransfer>
        <fefunca
          slope="0.1"
          type="linear"
        />
      </fecomponenttransfer>
    </filter>
    <circle
      cx="12"
      cy="12"
      r="10"
      style="filter: url(#EllipsisIcon__filter);"
    />
    <circle
      cx="6.5"
      cy="12"
      r="2"
    />
    <circle
      cx="12"
      cy="12"
      r="2"
    />
    <circle
      cx="17.5"
      cy="12"
      r="2"
    />
  </svg>
</div>
`;

exports[`icons <GeneralIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-GeneralIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M22.6607 9.27801C20.4579 10.703 19 13.1812 19 16C19 17.3216 19.3205 18.5684 19.888 19.6668C20.4176 19.122 20.8913 18.5226 21.2997 17.8777C21.1053 17.287 21 16.6558 21 16C21 14.2276 21.7685 12.6348 22.9905 11.5364C22.9583 10.7601 22.8456 10.0046 22.6607 9.27801Z"
      fill="currentColor"
    />
    <path
      d="M16.602 21.994C15.5831 20.2303 15 18.1833 15 16C15 11.5725 17.3978 7.70503 20.9657 5.62533C21.3515 6.16694 21.6894 6.745 21.9733 7.35333C18.9991 9.08615 17 12.3096 17 16C17 17.819 17.4857 19.5246 18.3345 20.9941C17.7912 21.3775 17.2117 21.7128 16.602 21.994Z"
      fill="currentColor"
    />
    <path
      d="M3.39698 5.14461C4.47981 5.69176 5.70393 5.99999 7.00005 5.99999C10.2635 5.99999 13.0705 4.04592 14.315 1.24401C13.6302 1.09734 12.9219 1.01441 12.1964 1.00171C11.1587 2.79417 9.22019 3.99999 7.00005 3.99999C6.25376 3.99999 5.53929 3.86374 4.88016 3.61476C4.33797 4.07559 3.84103 4.58807 3.39698 5.14461Z"
      fill="currentColor"
    />
    <path
      d="M18.0084 2.78435C16.1602 7.03088 11.9269 9.99999 7.00005 9.99999C5.02241 9.99999 3.15651 9.5216 1.51176 8.67422C1.7159 8.02985 1.97778 7.41116 2.29133 6.82424C3.69453 7.57457 5.29761 7.99999 7.00005 7.99999C11.1606 7.99999 14.7277 5.45915 16.2343 1.8445C16.8567 2.1043 17.4501 2.41965 18.0084 2.78435Z"
      fill="currentColor"
    />
    <path
      d="M9.91093 22.8019C9.33274 18.9519 6.01113 16 2.00005 16C1.91687 16 1.83398 16.0013 1.75141 16.0038C2.03454 16.728 2.39221 17.4148 2.81499 18.0549C5.11332 18.367 6.99803 19.9805 7.70253 22.1289C8.40431 22.427 9.14343 22.6543 9.91093 22.8019Z"
      fill="currentColor"
    />
    <path
      d="M1.00012 12.0411C1.32983 12.0139 1.66332 12 2.00005 12C8.23232 12 13.3549 16.751 13.9436 22.8288C13.3127 22.9413 12.6633 23 12 23C11.9836 23 11.9671 23 11.9507 22.9999C11.4489 17.9466 7.18536 14 2.00005 14C1.72648 14 1.45547 14.011 1.18745 14.0325C1.06682 13.3868 1.00261 12.7212 1.00012 12.0411Z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`icons <InProgressIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-InProgressIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <mask
      height="20"
      id="InProgressIcon__mask--0"
      mask-type="alpha"
      maskUnits="userSpaceOnUse"
      width="20"
      x="2"
      y="2"
    >
      <mask
        height="10"
        id="InProgressIcon__mask--1"
        maskUnits="userSpaceOnUse"
        width="20"
        x="2"
        y="12"
      >
        <path
          d="M12 22C17.5229 22 22 17.5229 22 12L2.00003 12C2.00003 17.5229 6.47718 22 12 22Z"
          fill="#232129"
        />
      </mask>
      <g
        mask="url(#InProgressIcon__mask--1)"
      >
        <path
          clip-rule="evenodd"
          d="M11.9996 22C17.5224 22 21.9995 17.5228 21.9995 12C21.9995 6.47714 17.5224 2 11.9996 2C6.47672 2 1.99957 6.47714 1.99957 12C1.99957 17.5228 6.47672 22 11.9996 22ZM11.9994 19.9999C16.4177 19.9999 19.9994 16.4182 19.9994 11.9999C19.9994 7.58166 16.4177 3.99994 11.9994 3.99994C7.58111 3.99994 3.99939 7.58166 3.99939 11.9999C3.99939 16.4182 7.58111 19.9999 11.9994 19.9999Z"
          fill="#2DE3DA"
          fill-rule="evenodd"
        />
      </g>
      <mask
        height="20"
        id="InProgressIcon__mask--2"
        maskUnits="userSpaceOnUse"
        width="20"
        x="2"
        y="2"
      >
        <path
          clip-rule="evenodd"
          d="M12 22C17.5228 22 22 17.5229 22 12C22 6.47718 17.5228 2.00003 12 2.00003C6.47714 2.00003 2 6.47718 2 12C2 17.5229 6.47714 22 12 22ZM11.9998 20C16.4181 20 19.9998 16.4182 19.9998 12C19.9998 7.58169 16.4181 3.99997 11.9998 3.99997C7.58154 3.99997 3.99982 7.58169 3.99982 12C3.99982 16.4182 7.58154 20 11.9998 20Z"
          fill="#2DE3DA"
          fill-rule="evenodd"
        />
      </mask>
      <g
        mask="url(#InProgressIcon__mask--2)"
      >
        <path
          d="M12 2.00003C6.47714 2.00003 2 6.47718 2 12L22 12C22 6.47718 17.5228 2.00003 12 2.00003Z"
          fill="url(#InProgressIcon__linear)"
        />
      </g>
    </mask>
    <g
      mask="url(#InProgressIcon__mask--0)"
    >
      <circle
        cx="12"
        cy="12"
        fill="currentColor"
        r="9.99998"
        style="filter: url(#InProgressIcon__filter);"
      />
    </g>
    <circle
      cx="12"
      cy="12"
      fill="currentColor"
      r="4"
    />
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="InProgressIcon__linear"
        x1="5.74999"
        x2="18.25"
        y1="12"
        y2="12"
      >
        <stop
          stop-color="#663399"
          stop-opacity="0"
        />
        <stop
          offset="1"
        />
      </lineargradient>
      <filter
        height="100%"
        id="InProgressIcon__filter"
        width="100%"
        x="0"
        y="0"
      >
        <fecomponenttransfer>
          <fefunca
            slope="0.3"
            type="linear"
          />
        </fecomponenttransfer>
      </filter>
    </defs>
  </svg>
</div>
`;

exports[`icons <IntegrationsIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-IntegrationsIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <circle
      r="2.125"
      stroke="currentColor"
      stroke-width="1.75"
      transform="matrix(1 0 0 -1 5.5 5.5)"
    />
    <circle
      r="2.125"
      stroke="currentColor"
      stroke-width="1.75"
      transform="matrix(1 0 0 -1 12 12)"
    />
    <circle
      fill="currentColor"
      r="2.75"
      transform="matrix(1 0 0 -1 18.75 18.5)"
    />
    <path
      d="M7.5 5.5H17.25C19.0449 5.5 20.5 6.95507 20.5 8.75V8.75C20.5 10.5449 19.0449 12 17.25 12H14"
      stroke="currentColor"
      stroke-width="2"
    />
    <path
      d="M14 18.5L6.75 18.5C4.95508 18.5 3.5 17.0449 3.5 15.25V15.25C3.5 13.4551 4.95507 12 6.75 12L10.5 12"
      stroke="currentColor"
      stroke-width="2"
    />
    <path
      d="M11.1517 15.8483L13.8033 18.5L11.1517 21.1517"
      stroke="currentColor"
      stroke-width="2"
    />
  </svg>
</div>
`;

exports[`icons <PortfolioIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-PortfolioIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M6.43308 15.749C6.06531 14.8383 5.92674 13.8512 6.02954 12.8744C6.13233 11.8976 6.47335 10.961 7.02265 10.1468C7.57196 9.33253 8.31275 8.66561 9.17999 8.20454C10.0472 7.74347 11.0144 7.50235 11.9966 7.50235C12.9788 7.50235 13.9459 7.74347 14.8132 8.20454C15.6804 8.66561 16.4212 9.33253 16.9705 10.1468C17.5198 10.961 17.8608 11.8976 17.9636 12.8744C18.0664 13.8512 17.9279 14.8383 17.5601 15.749"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M10.1551 7.78601L11.8461 10.048C11.9643 10.2056 12.0503 10.3849 12.0992 10.5757C12.1481 10.7666 12.1589 10.9651 12.131 11.1601C12.1032 11.3551 12.0372 11.5428 11.9368 11.7123C11.8364 11.8817 11.7037 12.0298 11.5461 12.148L9.74609 13.498V14.248C9.74609 14.6458 9.58806 15.0274 9.30675 15.3087C9.02545 15.59 8.64392 15.748 8.24609 15.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M17.629 11.427L15.323 12.587C15.0175 12.7396 14.7732 12.9918 14.6305 13.302C14.4878 13.6121 14.4551 13.9618 14.538 14.293L14.91 15.749"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M6.74609 2.24799V0.747986"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M17.2461 2.24799V0.747986"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M7.5 18.748C7.5 19.1458 7.65804 19.5273 7.93934 19.8086C8.22064 20.09 8.60218 20.248 9 20.248H15C15.3978 20.248 15.7794 20.09 16.0607 19.8086C16.342 19.5273 16.5 19.1458 16.5 18.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M22.2461 15.748H1.74609C1.19381 15.748 0.746094 16.1957 0.746094 16.748V22.248C0.746094 22.8003 1.19381 23.248 1.74609 23.248H22.2461C22.7984 23.248 23.2461 22.8003 23.2461 22.248V16.748C23.2461 16.1957 22.7984 15.748 22.2461 15.748Z"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M3 15.748V3.74799C3 3.35016 3.15804 2.96863 3.43934 2.68733C3.72064 2.40602 4.10218 2.24799 4.5 2.24799H19.5C19.8978 2.24799 20.2794 2.40602 20.5607 2.68733C20.842 2.96863 21 3.35016 21 3.74799V15.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;

exports[`icons <ReportsIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-ReportsIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <circle
      cx="12"
      cy="12"
      r="5"
      stroke="currentColor"
      stroke-width="2"
    />
    <rect
      fill="currentColor"
      height="2"
      width="4.5"
      x="17"
      y="4"
    />
    <rect
      fill="currentColor"
      height="2"
      width="2"
      x="4.5"
      y="6"
    />
    <rect
      fill="currentColor"
      height="2"
      width="4.5"
      x="15.5"
      y="18"
    />
    <path
      d="M3.5 11.5V20.5H12.5"
      stroke="currentColor"
      stroke-width="2"
    />
  </svg>
</div>
`;

exports[`icons <SkullIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-SkullIcon"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      clip-rule="evenodd"
      d="M12 2C16.4183 2 20 5.58172 20 10V14C20 15.7416 18.8869 17.2233 17.3333 17.7724V18C17.3333 20.2091 15.5425 22 13.3333 22H10.6667C8.45753 22 6.66667 20.2091 6.66667 18V17.7724C5.11308 17.2233 4 15.7416 4 14V10C4 5.58172 7.58172 2 12 2Z"
      fill="white"
      fill-rule="evenodd"
    />
    <rect
      fill="currentColor"
      height="3"
      rx="1.5"
      width="3"
      x="7"
      y="11"
    />
    <rect
      fill="currentColor"
      height="3"
      rx="1.5"
      width="3"
      x="14"
      y="11"
    />
    <line
      stroke="currentColor"
      stroke-width="1.5"
      x1="7"
      x2="17"
      y1="18.5"
      y2="18.5"
    />
    <mask
      fill="white"
      id="path-9-inside-1"
    >
      <path
        clip-rule="evenodd"
        d="M12 2C16.4183 2 20 5.58172 20 10V14C20 15.7416 18.8869 17.2233 17.3333 17.7724V18C17.3333 20.2091 15.5425 22 13.3333 22H10.6667C8.45753 22 6.66667 20.2091 6.66667 18V17.7724C5.11308 17.2233 4 15.7416 4 14V10C4 5.58172 7.58172 2 12 2Z"
        fill-rule="evenodd"
      />
    </mask>
    <path
      d="M17.3333 17.7724L16.6668 15.8867L15.3333 16.358V17.7724H17.3333ZM6.66667 17.7724H8.66667V16.358L7.33316 15.8867L6.66667 17.7724ZM22 10C22 4.47715 17.5228 0 12 0V4C15.3137 4 18 6.68629 18 10H22ZM22 14V10H18V14H22ZM17.9998 19.6581C20.3275 18.8354 22 16.6156 22 14H18C18 14.8677 17.4463 15.6112 16.6668 15.8867L17.9998 19.6581ZM15.3333 17.7724V18H19.3333V17.7724H15.3333ZM15.3333 18C15.3333 19.1046 14.4379 20 13.3333 20V24C16.647 24 19.3333 21.3137 19.3333 18H15.3333ZM13.3333 20H10.6667V24H13.3333V20ZM10.6667 20C9.5621 20 8.66667 19.1046 8.66667 18H4.66667C4.66667 21.3137 7.35296 24 10.6667 24V20ZM8.66667 18V17.7724H4.66667V18H8.66667ZM2 14C2 16.6156 3.67249 18.8354 6.00017 19.6581L7.33316 15.8867C6.55367 15.6112 6 14.8677 6 14H2ZM2 10V14H6V10H2ZM12 0C6.47715 0 2 4.47715 2 10H6C6 6.68629 8.68629 4 12 4V0Z"
      fill="currentColor"
      mask="url(#path-9-inside-1)"
    />
    <path
      d="M12 13L13.299 15.25H10.701L12 13Z"
      fill="currentColor"
      stroke="currentColor"
      stroke-width="0.5"
    />
  </svg>
</div>
`;

exports[`icons <SuccessIcon /> renders unchanged 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-testid="icon-SuccessIcon"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      clip-rule="evenodd"
      d="M22 12c0 5.5228-4.4772 10-10 10-5.52285 0-10-4.4772-10-10C2 6.47715 6.47715 2 12 2c5.5228 0 10 4.47715 10 10zm-11.3557 4.2175c-.2986-.0436-.5802-.2059-.76709-.4763l-2.48775-3.3963c-.25012-.362-.18368-.8548.15337-1.1376.33705-.2829.83396-.2627 1.14701.0465l1.76846 1.8456c.1645.1481.3187.207.5228.0394l4.7666-5.19716c.2935-.31996.7868-.35219 1.1194-.07311.3326.27907.3866.77052.1224 1.11509L11.788 15.7688c-.2786.3633-.724.5199-1.1437.4487z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
